hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/yaml/YamlProject.groovy

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.gradle.azkaban.yaml;
+
+/**
+ * Representation of a project in flow.project file for Azkaban Flow 2.0
+ */
+class YamlProject {
+  Double azkabanFlowVersion;
+
+  /**
+   * Constructor for YamlProject
+   */
+  YamlProject() {
+    azkabanFlowVersion = 2.0;
```

Comment:
Shall we define the constant "2.0" as a static final field?

Reply:
"static" isn't an allowed modifier for variables in Groovy. But I can make it final - I'll do so. Thanks!